### ISSUE TITLE
SteamCMD - app_run

### ISSUE THREAD

**Issue by alteredhero** [dgibbs64/SteamCMD-Commands-List#22]
what are the arguments I can use with this.

I'm currently attempting to change queryport from 27015 to 27016 when launching a specific app via steamcmd, but it doesn't seem to work, i can do it via a batch but i assume thats using launch parameters by actually launching the exe instead of how steamcmd launches an app. so it doesn't work with what I need to do. typically I would just change the settings within the ini file for the specific game, but for one of the specific games I'm working with, thats not an option, as they don't have the parameter set within the ini

simply put, attempting to use:
`app_run [gameid] -port 8212 -queryport 27016`
